packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-12/actions.test.js

```diff
@@ -27,33 +27,27 @@ test.before(async t => {
   t.context.bundleIds = await installBundles(bundlesData);
 });
 
-test.serial('Open Vaults', async t => {
+test.skip('Open Vaults', async t => {
```

Comment:
What exactly is failing in this test? It seems to be working on the current versions, no?

Reply:
Maybe the vaults count but I don't remember for certain.